homeassistant/components/romy/coordinator.py

```diff
@@ -0,0 +1,22 @@
+"""ROMY coordinator."""
+
+from romy import RomyRobot
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN, LOGGER, UPDATE_INTERVAL
+
+
+class RomyVacuumCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class RomyVacuumCoordinator(DataUpdateCoordinator[None]):
```
`DataUpdateCoordinator` is generic with the return type of `_async_update_data`.